src/customJs/socialShareTool/SocialShareTool.js

```diff
@@ -26,7 +30,7 @@ export const SocialShareTool = (props) => {
                   >
                     <img
                       width={values.social_share_size}
```

Comment:
I think that something is wrong here. 

`values.social_share_size` is the same that `size`, which means `small` or `big`. But, this value should be the half of image width to take advantage of retina displays.

This is the behavior in the old editor:

![image](https://user-images.githubusercontent.com/1157864/207352197-a7716f22-a02b-4eda-b36d-cb882479784a.png)

![image](https://user-images.githubusercontent.com/1157864/207352693-6ca54ea2-9648-4038-bad0-f9c6ab069aff.png)


![image](https://user-images.githubusercontent.com/1157864/207348658-4c3a87df-2f74-4f50-95a0-3dd068fe2bc2.png)

Reply:
I did use the width **_css_** because the with **HTML** no work for me.
![image](https://user-images.githubusercontent.com/6733401/209369772-2cf95ed0-9536-453f-beb8-3054de132b49.png)

Reply:
We talked about setting the width in both places in `width` property and in the `width` inline style.